task/3scale-toolbox-import/0.1/README.md

```diff
@@ -0,0 +1,138 @@
+# Maven
+
+This Task can be used to run the 3scale toolbox and  import to your 3scale API directly from a local OpenAPI spec compliant file or a remote URL.
+
+## Install the Task
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/task/3scale-toolbox-import/0.1/3scale-toolbox-import.yaml.yaml
+```
+
+## Parameters
+
+- **CONTEXT_DIR**: The path to the source
+- **PATH_OPENAPI**: The path to openapi file or url to be imported
+- **DESTINATION**: The name of 3scale target instance.
+- **PRIVATE_BASE_URL**: The URL of the API Backend, custom private base URL.
+- **TARGET_SYSTEM_NAME**: The Target system name.
+## Workspaces
+
+- **source**: `PersistentVolumeClaim`-type so that volume can be shared among `git-clone` and `import` task
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: 3scale-source-pvc
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 500Mi
+```
+
+## Platforms
+
+The Task can be run on `linux/amd64` platform.
```

Comment:
in the task it's mentioned several other platforms as well so can you please update here as well?

Reply:
Done